airflow/providers/google/cloud/operators/looker.py

```diff
@@ -0,0 +1,105 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""This module contains Google Cloud Looker operators."""
+
+from typing import Optional, Dict
+
+from airflow.models.baseoperator import BaseOperator
+from airflow.providers.google.cloud.hooks.looker import LookerHook
+
+
+class LookerStartPdtBuildOperator(BaseOperator):
```

Comment:
We are in the process to add Links to the GCP resources for most of the operators. Maybe this a good time to add this also to the Looker operators.

Example how you can do this you can find in the Dataproc Metastore: 
https://github.com/apache/airflow/pull/21267

Reply:
Got it! Is it the same as the [extra links](https://airflow.apache.org/docs/apache-airflow-providers/core-extensions/extra-links.html)?

Reply:
Exactly